tests/test-kubernetes.sh

```diff
@@ -52,9 +60,10 @@ kubectl_deploy() {
 
 verify_deploy(){
     echo "Verifying deployment was successful"
+    port=$(kubectl get svc -n istio-system istio-ingress -o jsonpath={.spec.ports[0].nodePort})
     IPS=$(bx cs workers "$CLUSTER_NAME" | awk '{ print $2 }' | grep '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')
     for IP in $IPS; do
-        if ! curl -sS http://"$IP":8080/version; then
+        if ! curl -sS http://"$IP":"$port"; then
```

Comment:
Where is `$port` coming from? If we are adding a new variable, please use a standard all uppercase variable name and be sure it is being set.